charts/incubator/lancache-monolithic/questions.yaml

```
@@ -81,7 +81,7 @@ questions:
                             description: "This port exposes the container port on the service"
                             schema:
                               type: int
-                              default: 10279
+                              default: 80
```

That's just the service port. And we won't allow 80 or 443 as default for any apps.
Because if someone installs it blindly it will lock him out of the Scale WebUI
```suggestion
                              default: 10279
```